spec/requests/atmosphere/api/appliance_configuration_templates_spec.rb

```diff
@@ -138,7 +138,8 @@
 
     context 'when unauthenticated' do
       it 'returns 401 Unauthorized error' do
-        post api("/appliance_configuration_templates")
+        post api("/appliance_configuration_templates"),
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.